src/config/services/index.md

```diff
@@ -32,3 +37,63 @@ Optionally, a service directory may contain:
 
 When a new service is created, a `supervise` folder will be automatically
 created on the first run.
+
+### Configuring services
+
+Most services can take configuration options set by a `conf` file in the service
+directory. This allows service customization without modifying the service
+directory provided by the relevant package.
+
+Check the service file for how to pass configuration parameters. Services
+usually expect `OPTS="--value ..."` in the `conf` file.
+
+To make more complex customizations than provided by default, [edit the
```

Comment:
i think we can remove "than provided by default".

Reply:
Indeed.